CMake/kwiver-depends-kpf.cmake

```diff
@@ -4,5 +4,8 @@ option( KWIVER_ENABLE_KPF
   "Enable kpf i/o for vital types"
   ${fletch_ENABLED_YAMLCPP}
 )
-# Mark this as advanced 
+# Mark this as advanced
 mark_as_advanced( KWIVER_ENABLE_KPF )
+if (KWIVER_ENABLE_KPF )
```

Comment:
Theoretically it could be possible to use KPF without YAML (if we add protobuf as an alternative transport). This particular patch enables the KPF-specific includes and whatnot; the find_package(YAML...) calls are in the per-directory CMakeLists.txt files with the vague idea that it made more sense to find YAML closest to where it was actually needed.

That said, if it's better / more customary to do that find_package once and localize all the searching and so forth here in the depends step, I'm happy to change it.